bundles/util/tools.vitruv.applications.util.temporary/build.properties

```diff
@@ -1,5 +1,4 @@
 source.. = src/,\
            xtend-gen/
 output.. = target/classes/
-bin.includes = META-INF/,\
-               .

```

Comment:
Is this intended? This will result in a plugin that only deploys metadata, won't it?

Reply:
No, this change was not intentional. I reverted it.

Reply:
Perfect, thanks!